docs/degree_programs_search_shortcode.md

```diff
@@ -12,6 +12,7 @@ See below for a list of supported attributes.
 | `hide`          | Comma-separated names of elements that should be hidden            | no       |         | See list below    |
 | `output`        | Output mode                                                        | no       | `list`  | `list` or `tiles` |
 | `filters`       | Comma-separated list of filters to be shown to visitors            | no       |         | See list below    |
+| `ids`           | Comma-separated list of HIS codes                                  | no       |         | See list below    |
```

Comment:
Even if we name it "HIS codes" in the source code, let's name it "Campo Keys" in all user-facing places.
```suggestion
| `ids`           | Comma-separated list of Campo Keys                                 | no       |         | See list below    |
```